strings/reverse_string.py

```diff
@@ -0,0 +1,26 @@
+def reverse(word: str) -> str:
+    """
+    Will reverse the entire string
+
+    >>> reverse("Hello World")
+    'dlroW olleH'
+    >>> reverse("wh[]32")
+    '23][hw'
+    >>> reverse("WHAT are you Doing?")
+    '?gnioD uoy era TAHW'
+    >>> reverse("gnirts tupni eht sesrever noitcnuf sihT")
+    'This function reverses the input string'
+    >>> reverse("")
+    ''
+    """
+
+    # creating a string slice that starts at the end of the string, and moves backwards.
+    # the slice statement [::-1] means start at the end of the string and end at position 0,
+    # move with the step negative one, which means one step backwards.
+    return "".join(word[::-1])
```

Comment:
You can just do return word[::-1] for this task.

Also, I feel like the point of this repository is to show the underlying implementations instead of using existing python ones.

Reply:
I did the same in my previous commit, but it was showing **Process completed with exit code 1 error**, that's why I added the join method but still, it's showing the same error.

okay, I am new here. I will surely keep that in mind from next time.